python/nav/web/static/js/src/webfront.js

```diff
@@ -222,7 +228,10 @@ require([
         setDefaultDashboardForm.submit(function (event) {
             event.preventDefault();
             feedback.removeAlertbox();
-            var request = $.post(this.getAttribute('action'));
+            const request = $.post(
```

Comment:
Why are you not sending a CSRF token here?

Reply:
What is happening here is that some jQuery magic is serializing the form, i.e. extracting the values and converting it to a request payload (with key=value pairs). As such, the code on line 233 `$(this).serialize()` makes sure that the CSRF token is sent with the request. 

This is how it works:
1. jQuery creates a reference to the form by using its ID, ie. `#form-set-default-dashboard`
2. On line 228 the `submit` event is called on the form, which triggers a submission.
3. Within the function callback, `this` refers to the form that just submitted.
4. As `this` contains the form values, it can be serialized to automatically include the `csrfmiddlewaretoken`.